### PR TITLE
makes weekly roundup + daily update emails sent via celery

### PR DESCRIPTION
**Pull request by owocki**
##### Description

makes weekly roundup + daily bounty emails sent via celery

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/6729

##### Testing

tested, works locally

WARNING: in a production environment, care should be taken to make sure that the 40k celery jobs created by each management command do not throttle the processing of other important jobs.  two ways we can do that:
1. create two celery "lane"s that define emails that run with expediency + emails that can be backlogged for a few hours
2. spin up enough celery servers that its just not an issue

will leave it to the engineers to decide which path is best.

### CODE REVIEW

File: app/marketing/mails.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def send_mail(from_email, _to_email, subject, body, html=False,
         mail.add_attachment(attachment)
 
     # debug logs
-    logger.info(f"-- Sending Mail '{subject}' to {to_email}")
+    logger.info(f"-- Sending Mail '{subject}' to {to_email.email}")
```
Comment (owocki) [inline app/marketing/mails.py:121]: not a direct celery related fix, but is super nice for readability

**Comment by danlipert** [comment]
@owocki I think the easiest way to create the lanes is to add rate limiting to the email tasks. We'll have to figure out the sweet spot so that the entire worker resources arent locked up. More info here: https://docs.celeryproject.org/en/stable/userguide/workers.html#rate-limits

**Comment by danlipert** [comment]
Marking as WIP so we don't merge until I spin up the new celery boxes and set up rate limiting

**Comment by owocki** [comment]
what settings / configuration r u thinking for the rate limiting? itd be interesting to have some boxes that rate limit some jobs, and others who rate limit other jobs.. effectively creating seperate queues. but i'd of course defer to u on the architecture.

**Comment by owocki** [comment]
spun up new celery servers, gonna merge this into master via  direct commit now

**Comment by owocki** [comment]
708165ec9597bb5ed1d629ddf992c06e34a0b61e

**Pull request closed without merging**